crates/lovely-win/build.rs

```diff
@@ -1,4 +1,5 @@
 fn main() {
+    #[cfg(target_os = "windows")]
```

Comment:
This can be removed as the binary crates (`-win`, `-mac`) are assumed to be built via `cargo b --target ... --package lovely-mac|win`.

Reply:
the forward-dll crate cannot be built on Mac, so when cargo attempts to setup the project, it always fails unless that's there. I couldn't find another way to fix it.

Reply:
Gotcha, weird that the build script is firing even if it's not explicitly targeted.